src/styleguide/_patterns/_generated/symbols.mustache

```diff
@@ -0,0 +1,11 @@
+<svg style="display: none;" aria-hidden="true" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Oh. This file shouldn't be checked into git.

It gets generated by the Pattern Lab build. However, as part of the recent Nunjucks conversion, the `.gitignore` rules was also updated. This is probably a left over from builds you did while we still used Mustache.

Please remove it.